scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -2221,13 +2225,14 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
           if (token.is[At] || (token.is[Ellipsis] && ahead(token.is[At]))) {
             t = atPos(t, auto)(Term.Annotate(t, annots(skipNewLines = false)))
           } else if (token.is[Underscore] && ahead(isStar)) {
-            if (allowRepeated) t = atPos(t, auto)({ next(); next(); Term.Repeated(t) })
-            else syntaxError("repeated argument not allowed here", at = token)
+            repeatedTerm(() => nextTwice())
           } else {
             // this does not necessarily correspond to syntax, but is necessary to accept lambdas
             // check out the `if (token.is[RightArrow]) { ... }` block below
             t = atPos(t, auto)(Term.Ascribe(t, typeOrInfixType(location)))
           }
+        } else if (dialect.allowPostfixStarVarargSplices && isStar) {
```

Comment:
This clause affects all `*` usage.
The community test has a lot of new similar errors:
```
Failed for file ..community-projects/dotty/compiler/src/scala/quoted/runtime/impl/printers/SourceCode.scala
Error: ..community-projects/dotty/compiler/src/scala/quoted/runtime/impl/printers/SourceCode.scala:103: error: ) expected but identifier found
    private def lineBreak(): String = "\n" + ("  " * indent)
```

I'm not sure maybe it should be  `dialect.allowPostfixStarVarargSplices && token.is[Ident] && ahead(isStar)` ?

Reply:
Damn, I will take a look into this. Martin's small changes are actually much more complicated than anticipated.

Reply:
Fix is here: https://github.com/scalameta/scalameta/pull/2240